content/en/docs/instrumentation/erlang/getting-started/phoenix.md

```diff
@@ -0,0 +1,271 @@
+---
+title: Phoenix
+weight: 2
+---
+
+This guide will show you how to get started with OpenTelemetry in the Phoenix
+Web Framework.
+
+You will learn how you can instrument a simple application automatically, in
+such a way that traces are emitted to the console.
+
+## Prerequisites
+
+Ensure that you have erlang, elixir, postgres (or the database of your choice),
+and phoenix installed locally. The phoenix
+[installation guide](https://hexdocs.pm/phoenix/installation.html) will help you
+get set up with everything you need.
+
+## Example Application
+
+The following example uses a basic [Phoenix](https://www.phoenixframework.org/)
+web application. For more elaborate examples, see
+[examples](/docs/instrumentation/erlang/examples/).
+
+To begin, use the `phx` generator to create a new project
+
+```shell
+mix phx.new your_app_name
+```
+
+Press `n` when asked to fetch and install dependencies, we'll do that after the
+next step.
+
+ℹ️ Throughout this guide I will use `your_app_name` or `YourAppName`, be sure to
+replace these with your actual application's name.
+
+### Dependencies
+
+We'll need a few other dependencies that Phoenix doesn't come with as well.
+
+- `opentelemetry_api`: contains the interfaces you'll use to instrument your
+  code. Things like `Tracer.with_span` and `Tracer.set_attribute` are defined
+  here.
+- `opentelemetry`: contains the SDK that implements the interfaces defined in
+  the API. Without it, all the functions in the API are no-ops.
+- `opentelemetry_exporter`: allows you to send your telemetry data to an
+  OpenTelemetry Collector and/or to self-hosted or commercial services.
+- `opentelemetry_ecto`: creates OpenTelemetry spans from the Elixir `:telemetry`
```

Comment:
Ecto should be left out from the example since it isn't used. We want this to start without a database. But since it is likely any user will have Ecto it should definitely include a link and mention of `opentelemetry_ecto`.

Should also add a link to the demo, https://github.com/open-telemetry/opentelemetry-demo/tree/main/src/featureflagservice, and examples dir of contrib, https://github.com/open-telemetry/opentelemetry-erlang-contrib/tree/main/examples/, saying something like, 'for more complete Phoenix/Ecto projects see these'.

Reply:
Should the example be explicitly created with `--no-ecto`?

Reply:
Yea.